src/views/products.vue

```diff
@@ -33,8 +33,7 @@
           </ion-thumbnail>
           <ion-label>
             <h2>{{ getProduct(product.groupValue).productName}}</h2>
-            <p v-if="$filters.getFeatures(getProduct(product.groupValue).featureHierarchy, '1/COLOR/')">{{ $t("Colors") }} : {{ $filters.getFeatures(getProduct(product.groupValue).featureHierarchy, '1/COLOR/') }} </p>
-            <p v-if="$filters.getFeatures(getProduct(product.groupValue).featureHierarchy, '1/SIZE/')">{{ $t("Sizes") }} : {{ $filters.getFeatures(getProduct(product.groupValue).featureHierarchy, '1/SIZE/') }} </p>
+            <p v-for="(attribute, index) in ($filters.groupFeatures(getProduct(product.groupValue).featureHierarchy))" :key="attribute" ><span class="features">{{ index }}</span>: {{ attribute }}</p>
```

Comment:
I think we have changed the markup. 
@azkyakhan @dt2patel 
Please review

Reply:
Also, improve variable name `index`

Reply:
Can you please tell me what was the need to use `span` tag and `features` class over here?